tests/test_stablediffusion.py

```diff
@@ -115,13 +130,17 @@ def prepare_inputs_for_testing(self):
                 "model_ids": PEFT_DIFFUSERS_SD_MODELS_TO_TEST,
                 "lora_kwargs": {"init_lora_weights": [False]},
                 "loha_kwargs": {"init_weights": [False]},
+                "oft_kwargs": {"init_weights": [False]},
             },
         )
     )
     def test_merge_layers(self, test_name, model_id, config_cls, config_kwargs):
         if config_cls == LoHaConfig:
             # TODO: This test is flaky with PyTorch 2.1 on Windows, we need to figure out what is going on
             self.skipTest("LoHaConfig test is flaky")
+        if config_cls == OFTConfig:
+            # TODO: This test is flaky with PyTorch 2.1 on Windows, we need to figure out what is going on
+            self.skipTest("OFTConfig test is flaky")
```

Comment:
I ran the test a couple of times and it passed on my Linux machine. I don't have Windows to test it. How big is the difference, are we talking about a very small deviation or something like completely random results? Does setting a seed help with flakiness?

If nothing helps and the deviation on Windows is very small, we could increase the tolerance a bit. Otherwise, let's skip the test on Windows.